### ISSUE TITLE
Add possibility to manually select Extraction zones

### ISSUE THREAD

**Issue by simontorres** [simontorres/goodman#41]
In some very special cases there might happen that the automatic target identification and extraction will not work properly, so there should exist the possibility to, by parsing an argument (most likely), manually select the regions for the target and for background subtraction.
This could be particularly helpful for targets that are too close or too close to the border (nobody should put their science target near the border though).

**Comment by b1quint** [comment]
Isn't this related to #1?

**Comment by simontorres** [comment]
Absolutely, but I think this can be treated independently because one thing is to improve the target detection (automatically) and another is to add a manual option (similar to the interactive mode for wavelength calibration). Despite this I would consider merging both issues. By the way, this came out from the discussion after the talk in the SOAR 2020 Workshop.

**Comment by b1quint** [comment]
This is one of the reasons that it has to be considered for the object extraction: if an input from the user may be an option, we have to have the code written to make this available in the future. If not, it may be too entangled so adding this new capability will be much more difficult.

**Comment by cbaorion** [comment]
Yes, we **definitely** want this feature!  Simon should code things so that this can be added later, but I would not include it for the Beta release.

**Comment by simontorres** [comment]
It will require some time, since we need to program a graphic user interface for this but if we are planning to create a special library for graphic treatment (see #42 )we better leave the effort for then.

**Comment by simontorres** [comment]
I'm currently having this issue with data that contains an extended source. Usually the background subtraction zone is selected an n-sigmas apart from the edge of the target extraction zone. What usually works for point sources does not work for extended sources Such as a [planetary nebula](http://simbad.u-strasbg.fr/simbad/sim-id?Ident=ic2448&NbIdent=1&Radius=2&Radius.unit=arcmin&submit=submit+id) (LEFT)
![background-extended-source](https://user-images.githubusercontent.com/3878895/36450700-8bf85464-166d-11e8-892b-60fb2d8e683a.png)
At least there should be a command line argument that allows to adjust the separation of the background zone.

**Comment by simontorres** [comment]
Could this be a solution for #13 ?
Not for fully automatic.

**Comment by simontorres** [comment]
related #298

**Comment by simontorres** [comment]
Upon review with @Allonck we think that despite this issue being very old it is still a desired feature in 2024.
